src/ResourceManager/Websites/Commands.Websites/Cmdlets/WebApps/SetAzureWebApp.cs

```diff
@@ -96,11 +96,18 @@ public class SetAzureWebAppCmdlet : WebAppBaseCmdlet
 
         [Parameter(Mandatory = false, HelpMessage = "Run cmdlet in the background")]
         public SwitchParameter AsJob { get; set; }
+        
+        [Parameter(ParameterSetName = ParameterSet1Name, Mandatory = false, HelpMessage = "Enable MSI on an existing azure webapp or functionapp")]
+        public bool AssignIdentity { get; set; }
```

Comment:
@panchagnula this should be a `SwitchParameter` instead of a `bool`. If the user wants to enable MSI, then they just provide the `-AssignIdentity` parameter, and if they don't, they exclude the parameter